src/Statement.php

```diff
@@ -174,10 +174,8 @@ public function sign($privateKeyFile, $privateKeyPass, $options = array()) {
                 if (! $cert) {
                     throw new \Exception('Unable to read certificate for x5c inclusion: ' . openssl_error_string());
                 }
-
-                if (! openssl_x509_export($cert, $x5c, true)) {
-                    throw new \Exception('Unable to export certificate for x5c inclusion: ' . openssl_error_string());
-                }
+                
```

Comment:
But if it is invalid it returns false so we have to capture the return value of the function call and then check it, at least according to http://php.net/manual/en/function.openssl-x509-export.php?